src/main/java/org/pac4j/jax/rs/annotations/Pac4JSecurity.java

```diff
@@ -0,0 +1,42 @@
+package org.pac4j.jax.rs.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.pac4j.jax.rs.filter.SecurityFilter;
+
+/**
+ * 
+ * Identify the class or method as being filtered by {@link SecurityFilter}.
+ * 
+ * @author Victor Noel - Linagora
+ *
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Pac4JSecurity {
+
+    /**
+     * @return value for {@link SecurityFilter#setAuthorizers(String)}
+     */
+    String[] value() default {};
```

Comment:
It is converted in a String later, but I prefer it to be better typed at that level.

By the way, there is no real reason to use a String with clients separated by commas except because you expect people to write a configuration file in the context of servlets. I wonder if it wouldn't be better if the logics were expecting arrays of string or list of strings, and the J2E servlet filters would make the conversion instead of imposing it to every user of the default logics.